doc/cabal-package.rst

```diff
@@ -1477,8 +1477,22 @@ system-dependent values for these fields.
 
     Version constraints use the operators ``==, >=, >, <, <=`` and a
     version number. Multiple constraints can be combined using ``&&`` or
-    ``||``. If no version constraint is specified, any version is
-    assumed to be acceptable. For example:
+    ``||``.
+    
+    .. Note::
+
+       Even though there is no ``/=`` operator, by combining operators we can
+       skip over one or more versions, to skip a deprecated version or to skip
+       versions that upset the constraint solving.
```

Comment:
"upset" is too informal to my taste.

Reply:
I'd suggest "complicate" or possibly "break".

Reply:
To ["throw into disorder"](https://www.merriam-webster.com/dictionary/upset) is one of the meanings of upset. I had used "overly narrow" in an early draft.

Reply:
I went with "to skip versions that narrow the constraint solving more than we'd like".